src/Components/GEOSldas_GridComp/Shared/my_matrix_functions.F90

```diff
@@ -15,6 +15,7 @@ module my_matrix_functions
   !public :: adjust_mean
   !public :: matrix_std
   public :: unique_rows_3col
+  public :: unique_rows_2col
```

Comment:
@weiyuan-jiang : After you are done with testing the subroutines, we should comment out this line again or enclose it within a pre-processor if statement (i.e., `#if 0 ... #endif` before we merge the branch.  The subroutine unique_rows_2col() is not used anywhere else in the code (besides the unit test program).

Reply:
> @weiyuan-jiang : After you are done with testing the subroutines, we should comment out this line again or enclose it within a pre-processor if statement (i.e., `#if 0 ... #endif` before we merge the branch. The subroutine unique_rows_2col() is not used anywhere else in the code (besides the unit test program).

I commented it out